curriculum/challenges/english/20-upcoming-python/learn-algorithm-design-by-building-the-shortest-path-algorithm/655771d889132f9ccd341060.md

```diff
@@ -0,0 +1,73 @@
+---
+id: 655771d889132f9ccd341060
+title: Step 5
+challengeType: 20
+dashedName: step-5
+---
+
+# --description--
+
+A graph is called a *weighted* graph when its edges are associated with weights, representing a distance, time or other.
+
+In your case, these weights will be the distances between each node, or point in space.
+To represent a weighted graph you can modify your dictionary, using a list of tuples for each value.
+The first element in the tuple will be the connected node, and the second element will be the distance.
+
```

Comment:
It would be better to give the syntax.

Here is an example:

```py
my_graph = {
	'A': [('B', weight-from-A-B), ('D', weight-from-A-D)],
.
.
}
```

Reply:
We should find another way. We should avoid text inside the examples.

Otherwise we can mention the text and explain its meaning so the translation can be done. In this case the description is already too long.